app/services/people/importer.rb

```diff
@@ -15,9 +15,10 @@ def initialize(nomis_attributes)
     end
 
     def call
-      person = Person.find_or_create_by!(nomis_prison_number: nomis_attributes[:prison_number])
+      person = Person.find_or_initialize_by(nomis_prison_number: nomis_attributes[:prison_number])
```

Comment:
What's the intention behind initialising instead of creating? If I'm reading this correctly, an effect of this change is that the returned `person` may or may not be persisted, which may not be obvious to consumers.

Reply:
There is only 1 consumer. These 'services' are less than ideal as the contract between them is less than obvious. Yes I've changed the contract as I wanted to do all the persistence in the same place (and know when we had created a new perosn record)